raaghu-components/src/rds-comp-forgot-password/rds-comp-forgot-password.tsx

```
@@ -107,7 +107,15 @@ const RdsCompForgotPassword = (props: RdsForgotPasswordProps) => {
                         </div>
                     </div>
                     </div>
-                        <div className="fs-6 d-inline-flex"><RdsLabel label="A password reset link will be sent to your email to reset your password. If you don't get an email in a few minutes, please re-try." size="13px"></RdsLabel></div>
+                        <div className="fs-6">
+                            <label>
+                            <RdsLabel 
```

ESLint

[error] Expected indentation of 32 spaces but found 28.

(indent)